package-lock.json

```diff
@@ -2251,6 +2251,11 @@
       "resolved": "https://registry.npmjs.org/plotly.js-dist/-/plotly.js-dist-1.57.1.tgz",
       "integrity": "sha512-tOOFxNc6PvdDkOUgWCSYc+CcxQULTpaGTROxrtmmF8NuZghcT3EInsToEzhPH3PcGM6bu61fKo73ZiFqxwYtlg=="
     },
+    "plotly.js-dist": {
+      "version": "1.58.4",
+      "resolved": "https://registry.npmjs.org/plotly.js-dist/-/plotly.js-dist-1.58.4.tgz",
+      "integrity": "sha512-oXCTRJFN8FBsHZSQPYoM3LuJQchPUrf6sOXFC0EFdvcr5lmJmLcAsW74jDy9PkRpm3PB+A+2oY1hsUMmk2eZbw=="
+    },
```

Comment:
Why is this change included without the changes of package.json?

Reply:
Previously, `plotly.js-basic-dist` was being used. However, it did not support contour plots, so I installed `plotly.js-dist` without realising that this particular change had been made in https://github.com/optuna/optuna-dashboard/commit/f8e35c19f715892e5650eb79407801c2ee10bfc4. While resolving the merge conflicts, I forgot to update this. 
I'll make the required change.